internal/generator/vector/conf/complex.toml

```diff
@@ -128,8 +128,8 @@ type = "kubernetes_logs"
 max_read_bytes = 3145728
 glob_minimum_cooldown_ms = 15000
 auto_partial_merge = true
-include_paths_glob_patterns = ["/var/log/pods/default_*/*/*.log", "/var/log/pods/kube*_*/*/*.log", "/var/log/pods/openshift*_*/*/*.log"]
-exclude_paths_glob_patterns = ["/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp", "/var/log/pods/openshift-logging_*/gateway/*.log", "/var/log/pods/openshift-logging_*/loki*/*.log", "/var/log/pods/openshift-logging_*/opa/*.log", "/var/log/pods/openshift-logging_elasticsearch-*/*/*.log", "/var/log/pods/openshift-logging_kibana-*/*/*.log", "/var/log/pods/openshift-logging_logfilesmetricexporter-*/*/*.log"]
+include_paths_glob_patterns = ["/var/log/pods/default_*/*/*", "/var/log/pods/kube*_*/*/*", "/var/log/pods/openshift*_*/*/*"]
```

Comment:
I have concerns this changes behavior.  This says..."read anything in these directories" but we only wish to explicitly read the "[0-9]*.log" files.  We also do not intend to read rolled over files since that implies we may read them twice.    "*/*/*" is appropriate IMO for excludes but not includes

Reply:
Agreed, if all we want is to exclude rotated files. I read the ticket differently, but see it now.  thanks.  Will make the change and let you know.